src/pages/AddItem/index.js

```diff
@@ -20,27 +24,41 @@ export const AddItemPage = () => {
   const onSubmit = (event) => {
     event.preventDefault();
     setIsLoading(true);
-
-    addProduct(formState)
-      .then((res) => console.log(res))
-      .catch(() => setError(true))
-      .finally(() => {
-        setIsLoading(false);
-
-        setFormState({
-          productName: '',
-          timeFrame: '7',
-          lastPurchaseDate: null,
+    console.log(products);
+    /*
+    Checks the state if the product already exists
+    Find returns undefined if it has not matches
+    */
+    const normalizer = (input) => input.toUpperCase().replaceAll(/[.,:;]/g, '');
```

Comment:
I really liked this function! Maybe this function could be in the Helpers folder

Reply:
I agree with Isabella 👍 . You can move this function or other independent functions to an external file 🙏 .